### PR TITLE
Forgotten Realms & Outland Patch Update

### PR DESCRIPTION
**Pull request by neronix17**
## Additions

- Support for Outland - Redburn Pact

## Changes

- Target mod rename for Forgotten Realms, Outland - Core and Outland - Eastborn Empire

## Reasoning

- Implemented in line with other existing patches

## Testing

Check tests you have performed:
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [x] Playtested a colony (long enough)

### CODE REVIEW

File: Patches/O21 Outland - Redburn Pact/Redburn_WeaponsMelee.xml [hunk 1]
```diff
@@ -0,0 +1,478 @@
...
+								<cooldownTime>1.59</cooldownTime>
+								<chanceFactor>0.10</chanceFactor>
+								<armorPenetrationBlunt>0.625</armorPenetrationBlunt>
+								<linkedBodyPartsGroup>Handle</linkedBodyPartsGroup>
+							</li>
+						</tools>
+					</value>
+				</li>
+
+				<!-- Naginata -->
```
Comment (N7Huntsman) [inline Patches/O21 Outland - Redburn Pact/Redburn_WeaponsMelee.xml:412]: I can't find a Naginata in the Redburn Pact mod. Maybe this is just a leftover and can be deleted?

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
Generally looks good, aside from a small error I'm getting for one of the weapons.

File: Patches/O21 Outland - Redburn Pact/Redburn_WeaponsMelee.xml (hunk 1 above)
Comment (neronix17) [reply Patches/O21 Outland - Redburn Pact/Redburn_WeaponsMelee.xml:412]: Ah yeah I must have pasted twice, was using it as a base to work from, Naginata is in Eastborn Empire so I probably had both loaded when testing it as I didn't spot the error.

Comment (N7Huntsman) [reply Patches/O21 Outland - Redburn Pact/Redburn_WeaponsMelee.xml:412]: That's sort of what I figured was the case--it appeared as a duplicate xml error initially and confused me at first. Are you able to make a new commit to remove those couple of operations? It'd probably be quicker than me opening my own PR against your branch.

Comment (neronix17) [reply Patches/O21 Outland - Redburn Pact/Redburn_WeaponsMelee.xml]: Yep that should be sorted now.

**Review by N7Huntsman — APPROVED** [review]
Looks good! Thanks for the contribution!

**Pull request merged**